index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<!--This means that the browser will (probably) render the width of the page at the width of its own screen-->
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Portafolio Page</title>
+	<!-- Call File CSS-->
+	<link rel="stylesheet" href="css/style.css">
+	<!-- Latest compiled and minified CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+</head>
+<body>
+	<div class="container">
+	<!-- Header that contains the Logo and Main Title -->
+		<header class="header">
+			<div class="Logo-Responsive">
+				<img class="img-responsive" src="Images/LogoUdacity.png" alt="Logo">
+			</div>
+			<div class="header-text">
+				<h1 class="header-title">Marcos Rodriguez</h1>
+				<h4 class="header-subtitle">Frontend Ninja</h4>
+			</div>			
+		</header>
+		<!-- Figure that contains the dividing line in the webpage -->
+		<figure class="figure-line">
+			<hr>
+		</figure>
+		<!-- Figure that contains the center image-->
+        <figure>
+			<img class="Image-Technology" src="http://www.masquenegocio.com/wp-content/uploads/2014/07/tecnolog%C3%ADa.jpg" alt="Technology">
+		</figure>
+		<!-- Section of featured work -->
+		<section class="featured-work">
+			<div class="featured-title">
+				<h2>Featured Work</h2>
+			</div>
+			<div class="Size-Images">
+				<img src="Images/process.jpg" alt="Image-Processor">
+				<h3 class="Subtitle">Processor</h3>
+					<a class="link-text" href="https://github.com/MRodriguezz?tab=repositories">Visit My Github</a>
+			</div>
+			<div class="Size-Images">
+				<img src="Images/Connecting.jpg" alt="Software">
+				<h3 class="Subtitle">Connecting</h3>
+				<a class="link-text" href="https://github.com/MRodriguezz?tab=repositories">Visit My Github</a>
+			</div>
+			<div class="Size-Images">
+				<img src="Images/SoftwareImage.jpg" alt="Software">
+				<h3 class="Subtitle">Software</h3>
+				<a class="link-text" href="https://github.com/MRodriguezz?tab=repositories">Visit My Github</a>
+			</div>
+		</section>
+	</div>
+</body>
```

Comment:
#### Awesome :white_check_mark: 

Your code looks very clean and easy to read, thanks!. Using documentation and style consistency, really makes a huge difference in the code readability.